tjpcov/covariance_fourier_ssc.py

```diff
@@ -104,7 +104,11 @@ def get_covariance_block(
         # Use the a's in the pk spline
         na = ccl.ccllib.get_pk_spline_na(cosmo.cosmo)
         a, _ = ccl.ccllib.get_pk_spline_a(cosmo.cosmo, na, 0)
-        a = a[1 / a < z_max + 1]
+        # Cut the array for efficiency
+        sel = 1 / a < z_max + 1
```

Comment:
What is sel besides a mask?  Maybe we more descriptive variable name could help?